tests/common.py

```diff
@@ -145,18 +146,58 @@ def post_model_outputs_and_maybe_allow_retries(
     stub: service_pb2_grpc.V2Stub,
     request: service_pb2.PostModelOutputsRequest,
     metadata: Tuple,
+    retryable_output_codes: List[int] = [],
```

Comment:
It's probably better to just have a default list of status codes for all reqs

Reply:
I think the caller should be explicit about what codes it should retry on.

Reply:
That means we're special casing just the image tests, which doesn't seem like it should have special treatment if we'd just be adding this for other tests when they start failing.
If we don't want to hide failures, we still are here by passing the same retry-able status codes for all image predict tests.

Reply:
I can set the default to `INTERNAL_UNCATEGORIZED` to solve the staging issue, but we will potentially retry in cases we don't mean to.

Reply:
Right, we should fix the internal handling of this failure case to return a more concrete status code

Reply:
I set a default retryable code for all invocations of `post_model_outputs_and_maybe_allow_retries` to have more consistent behavior, but it sounds like we want to fix the Go API status code returned by this request first, then?

Reply:
lets fix here -> then API -> then come back to this fix